draft-jones-oauth-rfc7523bis.xml

```diff
@@ -265,20 +267,26 @@
 	  </t>
 	  <t>
 	    The JWT MUST contain an <spanx style="verb">aud</spanx>
-	    (audience) claim containing a value that
-	    identifies the authorization server as an intended audience.
-	    The token endpoint URL of the authorization server MAY be used as a
-	    value for an <spanx style="verb">aud</spanx> element to identify
-	    the authorization server as an intended audience of the JWT.
-      The authorization server MUST reject any JWT that does not
-      contain its own identity as the intended audience.
-      In the absence of an application profile specifying
-      otherwise, compliant applications MUST compare the audience
-      values using the Simple String Comparison method defined in Section
-      6.2.1 of RFC 3986 <xref target="RFC3986"/>.
-      As noted in <xref target="Interoperability"/>, the precise strings to be used
-      as the audience for a given authorization server must be configured out of band
-      by the authorization server and the issuer of the JWT.
+	    (audience) claim containing
+	    the issuer identifier <xref target="RFC8414"/>
+	    of the authorization server as its sole value.
+	    The authorization server MUST have an issuer identifier
```

Comment:
There are still valid uses of this specification when an authorization server doesn't have an issuer identifier.

Reply:
Deployments not needing the security fix need make no changes.  They can continue using RFC 7523 as is.

We should aim to be very clear in our public description of the problem when deployments do and do not require updates.